acceptance/indy_acceptance/README.md

```diff
@@ -0,0 +1,61 @@
+
+
+# Basic Indy Acceptance Test
+
+This is Python Basic Acceptance test for Indy. The tests are not driven by any unit test framework but are standalone python scripts.
+
+This test currently requires python 3.6.
```

Comment:
python3-indy is compatible with Python 3.5 since the version 1.0.0-dev-171. Currently indy-node/acceptance uses python3-indy 1.0.0-dev-177. So Python 3.5 is required, not 3.6.

Reply:
@spivachuk We followed the first python3.6 instruction but the tests are able to run with python3.5. Will update the python requirement.